File: Task1/C++/main.cpp
#include "library.h"

int main() {
    // Задача 1
    // Инициализация матрицы и поиск строк с отрицательными элементами
    std::cout << std::string(30, '-') << std::endl;
    std::cout << "Задание 1" << std::endl;
    std::cout << std::string(30, '-') << std::endl;
    taskOne();
    std::cout << std::endl;

    // Задача 2
    // Работа с квадратной матрицей
    std::cout << std::string(30, '-') << std::endl;
    std::cout << "Задание 2" << std::endl;
    std::cout << std::string(30, '-') << std::endl;
    taskTwo();
    std::cout << std::endl;

    // Задача 3
    // Игра Жизнь
    std::cout << std::string(30, '-') << std::endl;
    std::cout << "Задание 3: Игра \"Жизнь\"" << std::endl;
    std::cout << std::string(30, '-') << std::endl;
    taskThree();

    return 0;
}

File: Task1/Java/Main.java
public class Main {

	public static void main(String[] args) {
		// Task 1
		System.out.println(new String(new char[30]).replace('\0', '-'));
		System.out.println("Задание 1");
		System.out.println(new String(new char[30]).replace('\0', '-'));
		MatrixOperations.taskOne();
		System.out.println();

		// Task 2
		System.out.println(new String(new char[30]).replace('\0', '-'));
		System.out.println("Задание 2");
		System.out.println(new String(new char[30]).replace('\0', '-'));
		SquareMatrixOperations.taskTwo();
		System.out.println();

		// Task 3
		System.out.println(new String(new char[30]).replace('\0', '-'));
		System.out.println("Задание 3: Игра \"Жизнь\"");
		System.out.println(new String(new char[30]).replace('\0', '-'));
		GameOfLife.taskThree();
	}
}
